README.md

```diff
@@ -1,12 +1,18 @@
-# Official Ruby-on-Rails Integration with Sass
+# Official Ruby on Rails Integration with Sass
+
+[![Travis CI](https://api.travis-ci.org/rails/sass-rails.svg)](http://travis-ci.org/rails/sass-rails)
+[![Gem Version](https://badge.fury.io/rb/sass-rails.svg)](http://badge.fury.io/rb/sass-rails)
+[![Dependencies](https://gemnasium.com/rails/sass-rails.svg)](https://gemnasium.com/rails/sass-rails)
 
 This gem provides official integration for Ruby on Rails projects with the Sass stylesheet language.
 
-## Installing
+## Installation
 
 Since Rails 3.1, new Rails projects will be already configured to use Sass. If you are upgrading to Rails 3.1 you will need to add the following to your Gemfile:
 
-    gem 'sass-rails'
+```
```

Comment:
This is nit-picky but the real advantage of fenced code blocks is to be able to specitfy the language ; could you add `ruby` after the three back-ticks please ?